source/upgrade.txt

```diff
@@ -0,0 +1,90 @@
+.. _entity-framework-upgrade-provider:
+
+========================
+Upgrade Provider Version
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: breaking change, update, version
+
+Overview
+--------
+
+In this section, you can identify essential changes you must make to your
+application when you upgrade your provider to a new version.
+
+Before you upgrade, perform the following actions:
+
+- Ensure the new version is compatible with the {+mdb-server+} version
+  your application connects to and the {+language-dotnet+} version your
+  application runs on. See the :ref:`Compatibility <entity-framework-compatibility>`
+  section for this information.
+- Address any breaking changes between the current version of the provider
+  your application is using and your planned upgrade version in the
+  :ref:`Breaking Changes <entity-framework-breaking-changes>` section.
+
+.. _entity-framework-breaking-changes:
+
+Breaking Changes
+----------------
+
+A breaking change is a modification of a convention or a behavior starting in
+a specific version of the provider. This type of change may prevent your application from
+working properly if not addressed before upgrading the provider.
+
+The breaking changes in this section are categorized by the provider version that
+introduced them. When upgrading provider versions, address all the breaking
+changes between the current and upgrade versions. For example, if you
+are upgrading the provider from v8.0 to v8.5, address all breaking changes from
+the version after v8.0 including any listed under v8.5.
+
+.. _entity-framework-breaking-changes-v8.1:
+
+Version 8.1.0 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Updated the {+provider-short+} to run all changes as MongoDB transactions. If your
+  application uses {+mdb-server+} earlier than v5.0, or does not run in a
+  replica set, you must make the following changes to your application:
+
+  - If your application is configured for {+mdb-server+} earlier than v5.0, you
+    must upgrade to {+mdb-server+} v5.0 or later.
+  - If your application runs on a standalone server, you must reconfigure your
+    standalone server to a single-instance replica set. To learn more about how
+    to reconfigure your standalone server, see
+    :manual:`Convert a Standalone Self-Managed mongod to a Replica Set </tutorial/convert-standalone-to-replica-set>` in the
+    {+mdb-server+} manual.
+  - If your application runs in a container environment, switch to a
+    single-instance replica set container if one is available.
+
+  To avoid snapshot issues during transactions, call the ``EnsureCreated()`` or
+  ``EnsureCreatedAsync()`` during your application start-up, rather than
+  creating the collection or database
+  implicitly.
+
+  If you don't want to ensure optimistic concurrency with transactions, you can
```

Comment:
"optimistic concurrency" is a new phrase to me, and i wonder if it might also be for our users. would a link to wikipedia or somewhere make sense here? 

not positive this syntax is correct:

```suggestion
  If you don't want to ensure :wikipedia:`optimistic concurrency <Optimistic_concurrency_control>` with transactions, you can
```